### ISSUE TITLE
add port knocking

### ISSUE THREAD

**Issue by adeck** [adeck/ansible-deployments#7]
...Using fwknopd for SPA. That's the idea, anyway.

**Comment by adeck** [comment]
Interestingly, not blocked by another issue. Blocked by the fact that, at the moment, `fwknopd` appears to require glibc 2.15, and Centos 6.5 is a glibc 2.12 O/S. Always has been, probably always will be.

But all is not lost!
Michael Rash promised to add Centos 6.5 support!

Incidentally, dude has fantastic response time to e-mail, and an all-around pleasant demeanor.
